tests/integration/config/plugins/integration-tests/data/tests/tests_common.txt

```diff
@@ -691,6 +691,30 @@ test "Depart"
 		call "Wait 55 steps"
 
 
+test "Depart and Ignore Warning Message"
+	status partial
+	description "Helper function to depart from a planet when a message requiring acknowledgement via keyboard input is expected."
+	sequence
+		input
+			key d
+		# The game needs 2 gamesteps for processing the launch before commands like the jump command can be given.
+		# Using empty inputs to trigger those game loops.
+		input
+		input
+		# Dismiss the expected message
```

Comment:
```suggestion
		# Acknowledge the expected message
```
dismissing it would be pressing escape wouldnt it?
this does ok so it accepts whatever is asked

mb rename the test too